com.github.d4nj1.tlpui.yml

```
@@ -0,0 +1,53 @@
+app-id: com.github.d4nj1.tlpui
+command: tlpui
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+finish-args:
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+  # Required for X11 fallback
+  - --share=ipc
+  - --socket=fallback-x11
+  # TLP config and defaults files
+  #- --filesystem=/etc
+  #- --filesystem=/usr
+  #- --filesystem=/var/run/host/etc
+  #- --filesystem=/var/run/host/usr
+  # As the above don't work we have to use host permission
+  - --filesystem=host
+  # TLP-UI config files (not needed due to host permissions)
+  #- --filesystem=xdg-config
+  #- --filesystem=~/.config/tlpui
+modules:
+  - name: tlpui
+    buildsystem: simple
+    build-commands:
+      - install -D host-exec-wrapper.sh /app/bin/host-exec-wrapper
+      - install -D tlpui/icons/themeable/hicolor/scalable/apps/tlpui.svg /app/share/icons/hicolor/scalable/apps/com.github.d4nj1.tlpui.svg
+      - install -D tlpui/icons/themeable/hicolor/16x16/apps/tlpui.png /app/share/icons/hicolor/16x16/apps/com.github.d4nj1.tlpui.png
+      - install -D tlpui/icons/themeable/hicolor/32x32/apps/tlpui.png /app/share/icons/hicolor/32x32/apps/com.github.d4nj1.tlpui.png
+      - install -D tlpui/icons/themeable/hicolor/48x48/apps/tlpui.png /app/share/icons/hicolor/48x48/apps/com.github.d4nj1.tlpui.png
+      - install -D tlpui/icons/themeable/hicolor/64x64/apps/tlpui.png /app/share/icons/hicolor/64x64/apps/com.github.d4nj1.tlpui.png
+      - install -D tlpui/icons/themeable/hicolor/96x96/apps/tlpui.png /app/share/icons/hicolor/96x96/apps/com.github.d4nj1.tlpui.png
+      - install -D tlpui/icons/themeable/hicolor/128x128/apps/tlpui.png /app/share/icons/hicolor/128x128/apps/com.github.d4nj1.tlpui.png
+      - install -D tlpui/icons/themeable/hicolor/256x256/apps/tlpui.png /app/share/icons/hicolor/256x256/apps/com.github.d4nj1.tlpui.png
+      - install -D AppImage/com.github.d4nj1.tlpui.appdata.xml /app/share/metainfo/com.github.d4nj1.tlpui.metainfo.xml
+      - sed -i 's/Icon=tlpui/Icon=com.github.d4nj1.tlpui/g' tlpui.desktop
```

```suggestion
      - desktop-file-edit --set-icon=com.github.d4nj1.tlpui tlpui.desktop
```

But you could as well had all properly named upstream.